Multisig/UI/UI Library/Views/ClaimableAmountView.swift

```diff
@@ -17,6 +17,10 @@ class ClaimableAmountView: UINibView {
         layer.borderWidth = 2
         layer.cornerRadius = 8
         clipsToBounds = true
+    }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
         layer.borderColor = UIColor.clear.cgColor
```

Comment:
because when changing ligth to dark mode, the cgColor is not updating otherwise. You need to update the border color in layout subviews so that it takes the dark mode color variant